### DISCUSSION TITLE
Invalid workflow file

### DISCUSSION THREAD

**Discussion by TOBY19k** [community/community#40438] [Actions] [answered]
I keep getting errors when I try to make work flow file that zips a folder without compressing it you can see the error [here](https://github.com/TOBY19k/Seal-Os-Boot-Animation-Installer/actions/runs/3583479830/workflow)

**Comment by airtower-luna** [comment, marked as answer]
The error message is misleading, unfortunately: The problem isn't on line 25 (that's just a valid comment), it's in the list (`steps`) that *starts* at line 25. The indentation in the [last item in your list](https://github.com/TOBY19k/Seal-Os-Boot-Animation-Installer/blob/ccf3e348acd1366635b6e10f627819e3feb6d6db/.github/workflows/blank.yml#L37-L40) (the `uses: actions/upload-artifact@v3` part) is broken:

* The `-` that starts the item must be at the same indentation level as the other items, there's one space too many in front of it.
* `with` must be at the same indentation level as `uses`.

In YAML indentation defines structure, so items in the same list or mapping must line up exactly.

**Reply by TOBY19k** [reply]
Ok thanks for the response 🙂

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
